pom.xml

```diff
@@ -3580,6 +3580,18 @@
         </os>
       </activation>
     </profile>
+    <profile>
+      <id>mac-on-apple-silicon</id>
+      <properties>
+        <test.exclude.tags>org.apache.spark.tags.ExtendedLevelDBTest,org.apache.spark.tags.ExtendedRocksDBTest</test.exclude.tags>
```

Comment:
Hm, I think we don't need to bother with this for now. Testing tags are enough. I would prefer to avoid having profiles here for other combinations of tests.

Reply:
Ok ~ So is it necessary for us to remind developers using M1 that they should manually add these tags when testing? I don't think all developers know these at present.

Reply:
@HyukjinKwon 7a17868 change to re-write `test.default.exclude.tags`, in this way, we can still use `test.exclude.tags` in other scenarios. Can this ease your concerns?

Reply:
WDYT @dongjoon-hyun ?

Reply:
> So is it necessary for us to remind developers using M1 that they should manually add these tags when testing?

If users want to set `-Dtest.default.exclude.tags` manually (e.g. for removing `ChromeUITest` tag), they still need to know about `Extended{LevelDB,RocksDB}Test` tags because the default value of `test.default.exclude.tags` set by the `mac-on-apple-silicon` profile is overridden.

So, another option would be to note how to test using Apple silicon on [the developer's doc](http://spark.apache.org/developer-tools.html).

Reply:
If the user really needs to customize `-Dtest.default.exclude.tags` instead of `-Dtest.exclude.tags`, it is better to modify the developer's doc